src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -106,6 +106,7 @@ export default class AircraftModel {
          */
         this.id = _uniqueId('aircraft-');
 
+        // TODO: Does the aircraft itself really need this?
```

Comment:
It did and I'd assume it still does.  But maybe not anymore?

Reply:
It apparently needs it for _instantiation_ of the FMS, but after passing it along, the aircraft itself _does not_ need it. Removed.